### ISSUE TITLE
Ctrl+R does not change the repo

### ISSUE THREAD

**Issue by damithc** [HubTurbo/HubTurbo#845]
I've never used this feature before. Was it working before?

**Comment by damithc** [comment]
Related:
* This feature is not documented. 
* We usually use `Ctrl` to mean 'new' (e.g. Ctrl+I is for 'new issue'). Perhaps `Alt`+R is a better choice, although I haven't seen `Alt` key used like that in other apps?.

**Comment by acruis** [comment]
<kbd>Ctrl+R</kbd> only works if a certain panel is focused.

Also, what about <kbd>Ctrl+W</kbd>? It closes a panel and doesn't create anything new.

**Comment by damithc** [comment]
It doesn't work in most cases even if a panel is in focus. I've tried about 20 times and it worked only once. So there is some hidden condition other than the panel focus. 
Yes I forgot `Ctrl+W`. I guess we can consider `Ctrl` to indicate 'add/remove' kind of actions. I don't mind using for 'switch' too, but the more diverse we make it, the harder it becomes to remember.

**Comment by acruis** [comment]
It works fine for me, both when focusing on the filter text field, having an issue selected, or just focusing on a panel itself. How many repos do you have in your cache?

**Comment by damithc** [comment]
OK, I figured out the problem. It works if I do it slow. i.e. press ctrl first, and then, press R. pressing them together doesn't seem to work. Not sure if it a problem with my keyboard or how I press the keys :-p

**Comment by damithc** [comment]
Closing now. Probably It's a problem on my side.
We can keep the Ctrl because we seem to use that key for many things already.
Need to document, but that can be a separate issue.

**Comment by acruis** [comment]
Yep, we're detecting the keys by seeing if <kbd>R</kbd> is released while <kbd>Ctrl</kbd> is held down :-) You probably released <kbd>Ctrl</kbd> before R was released.
